src/model/Guess.java

```diff
@@ -11,24 +11,24 @@
  * @author nathan
  */
 public class Guess {
-    public String name;
-    public String typing;
-    public String paradigm;
-    public String level;
-    public int year;
+    private String name;
+    private String typing;
+    private int paradigmID; // 1 = procedural, 2 = structured, 3 = object oriented, 4 = functional, 5 = logic
+    private String paradigmName;
+    private int year; 
     
     /**
      * Parameterized constructor for setting all the values at once
      * @param nameIn
      * @param typingIn
-     * @param paradigmIn
+     * @param paradigmIDIn
      * @param levelIn
      * @param yearIn 
      */
-    public Guess(String nameIn, String typingIn, String paradigmIn, String levelIn, int yearIn) {
+    public Guess(String nameIn, String typingIn, int paradigmIDIn, String levelIn, int yearIn) {
         name = nameIn;
         typing = typingIn;
-        paradigm = paradigmIn;
+        paradigmID = paradigmIDIn;
         level = levelIn;
         year = yearIn;
```

Comment:
I would change this Guess constructor to read in `paradigmName`, then just make a function to convert from the name to an ID, and call it in the constructor to set the ID.